src/sagemaker/content_types.py

```diff
@@ -10,12 +10,98 @@
+            specifications should be tolerated (exception not raised). If False, raises an
+            exception if the script used by this version of the model has dependencies with known
+            security vulnerabilities. (Default: False).
+        tolerate_deprecated_model (bool): True if deprecated models should be tolerated
+            (exception not raised). False if these models should raise an exception.
+            (Default: False).
+    Returns:
+        list: The supported content types to use for the model.
+
+    Raises:
+        ValueError: If the combination of arguments specified is not supported.
+    """
+    if not jumpstart_utils.is_jumpstart_model_input(model_id, model_version):
+        raise ValueError(
+            "Must specify `model_id` and `model_version` when retrieving content types."
+        )
+
+    return artifacts._retrieve_supported_content_types(
+        model_id,
+        model_version,
+        region,
+        tolerate_vulnerable_model,
+        tolerate_deprecated_model,
+    )
+
+
+def retrieve_default(
+    region: Optional[str] = None,
+    model_id: Optional[str] = None,
+    model_version: Optional[str] = None,
+    tolerate_vulnerable_model: bool = False,
+    tolerate_deprecated_model: bool = False,
+) -> str:
+    """Retrieves the default content type for the model matching the given arguments.
+
+    Args:
+        region (str): The AWS Region for which to retrieve the default content type.
+            Defaults to ``None``.
+        model_id (str): The model ID of the model for which to
+            retrieve the default content type. (Default: None).
+        model_version (str): The version of the model for which to retrieve the
+            default content type. (Default: None).
+        tolerate_vulnerable_model (bool): True if vulnerable versions of model
+            specifications should be tolerated (exception not raised). If False, raises an
+            exception if the script used by this version of the model has dependencies with known
+            security vulnerabilities. (Default: False).
+        tolerate_deprecated_model (bool): True if deprecated models should be tolerated
+            (exception not raised). False if these models should raise an exception.
+            (Default: False).
+    Returns:
+        str: The default content type to use for the model.
+
+    Raises:
+        ValueError: If the combination of arguments specified is not supported.
+    """
+    if not jumpstart_utils.is_jumpstart_model_input(model_id, model_version):
+        raise ValueError(
+            "Must specify `model_id` and `model_version` when retrieving content types."
+        )
+
+    return artifacts._retrieve_default_content_type(
+        model_id,
+        model_version,
+        region,
+        tolerate_vulnerable_model,
+        tolerate_deprecated_model,
+    )
 
-deprecations.removed_warning("The sagemaker.content_types module")
```

Comment:
If we are un-deprecating, could someone from the SDK team signoff?

Reply:
Also given that the functions are calling jumpstart utilities, should we just move to the `jumpstart` namespace?

Reply:
we want to keep the approach of using top-level utilities for getting jumpstart arts (model_uris, script_uris, content_types, etc.)

Reply:
What is the rationale? At a product level, I would think that we would prefer to keep all of our utilities namespaced under `jumpstart` to increase the brand recognition.

Reply:
For context Chris: there has been debate in the past about the JumpStart branding vs Built-in Algorithm. 

One of the issue is that we use the common, top-level utility [image_uris.retrieve](https://github.com/aws/sagemaker-python-sdk/blob/2bac010cffa615d624578be822db6e00c2826d4c/src/sagemaker/image_uris.py#L42).

This is the precedent we've been using for all other utilities such as [script](https://github.com/aws/sagemaker-python-sdk/blob/2bac010cffa615d624578be822db6e00c2826d4c/src/sagemaker/script_uris.py#L25), [artifacts](https://github.com/aws/sagemaker-python-sdk/blob/2bac010cffa615d624578be822db6e00c2826d4c/src/sagemaker/model_uris.py#L26) or [hyperparameters](https://github.com/aws/sagemaker-python-sdk/blob/2bac010cffa615d624578be822db6e00c2826d4c/src/sagemaker/hyperparameters.py#L28).